src/frontend/src/components/initialLogin/InitialLogin.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export function InitialLogin() {
+  return (
+    <h1>Please <strong><a href="">login</a></strong> first.</h1>
```

Comment:
2 dingen:

1. We gebruiken geen anchor tags in deze app. We maken gebruik van een library die het linken naar andere pagina's voor ons regelt: React Router. In `index.tsx` zie je dat onze hele applicatie omringt wordt door een `Router` component. Daar zie je ook dat dat er een `Route` component bestaat waarbij de `path` property `/login` is. Daaronder zie je als child het `Login` component. Dat betekent dat als de `path` in de url bar (de path is alles na het .com deel, bijv bij www.arthurpanteleev.com/login is /login de path) `/login` is, dat dan het `Login` component gerendered wordt.
React Router regelt het linken voor ons. We gebruiken dus niet zelf anchor tags, maar een comonent uit de React Router library: `Link`. 
Jouw opdracht: vervang die anchortag hier met een `Link` component.

2. Gezien er een route bestaat die het `Login` component regelt moet je in dat `Link` component uit 1. even uitzoeken hoe je linkt naar die route. Oftewel, als je als user klikt op dat component wil ik dat je naar de Login page gaat.

Do it :D 
PS: Lezen van de documentatie helpt.